contrib/caldav/calcurse-caldav.py

```diff
@@ -43,6 +43,12 @@ def die_atnode(msg, node):
         msg += etree.tostring(node).decode('utf-8')
     die(msg)
 
+def validate_sync_filter():
+    valid_sync_filter_values = {'event', 'apt', 'recur-event', 'recur-apt', 'todo', 'recur', 'cal'}
+
+    filter_items = set(sync_filter.split(','))
+
+    return filter_items - valid_sync_filter_values
```

Comment:
Small nit: I think this is easier to read if you make it a two-liner, initializing `valid_sync_filter_values` in the first line and computing the return value (i.e. `return set(sync_filter.split(',')) - valid_sync_filter_values`) in the second line, without an empty line inbetween.

Reply:
We also separate functions by two empty lines. So there is one empty line missing before the function and another one after the function.